components/automate-chef-io/habitat/plan.sh

```diff
@@ -6,6 +6,7 @@ pkg_license=('Chef-MLSA')
 pkg_description="HTML for automate.chef.io"
 pkg_upstream_url="https://www.chef.io/automate"
 pkg_build_deps=(
+  core/curl
```

Comment:
This was already pulled in as a transitive dep so this wasn't necessary, but it seemed better to be explicit.